woocommerce/woocommerce
# [Product Block Editor]: add Linked product sections. First approach.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Depens on https://github.com/woocommerce/woocommerce/pull/43009
Part of https://github.com/woocommerce/woocommerce/issues/42915
Closes https://github.com/woocommerce/woocommerce/issues/42915



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `product-linked` flag is enabled 
2. Go to the Product Editor
3. Select the `Linked Products` section
4. Confirm you see the `Upsell` and `Cross-sells` section
5. Confirm the text and link of the help tooltip

Upsells | Cross-sells
-----|------
<img width="739" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/a8a25be1-0676-4899-874b-944ac04ca5ec"> | <img width="705" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/b480cbe9-4767-417a-b209-043e6960b47e">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
